src/signals/incident/components/form/ContainerSelect/context.tsx

```diff
@@ -6,9 +6,11 @@ export const initialValue: ContainerSelectValue = {
   selection: [],
   location: [0, 0],
   meta: { endpoint: '', featureTypes: [] },
+  message: undefined,
   update: /* istanbul ignore next */ () => {},
   edit: /* istanbul ignore next */ () => {},
   close: /* istanbul ignore next */ () => {},
+  setMessage: /* istanbul ignore next */ () => {},
 };
```

Comment:
Not part of this PR, but we're providing a default context value that should never be used (empty functions). Instead, we could also write a `useContainerSelectContext` hook that throws an error when 'value' is not supplied. Example here: https://kentcdodds.com/blog/how-to-use-react-context-effectively#the-custom-consumer-hook

Reply:
Great suggestion! I had exactly this in mind for some time now :-)